client/logic/game/terrain/terrain.gd

```diff
@@ -100,8 +100,8 @@ func uncapture() -> void:
 
 
 func get_terrain_by_position(pos: Vector2i) -> Terrain:
-	# check if terrain lookup is generated (dictionary exist and entries aren't null). If not -> generate
-	if not (_terrain_lookup and _terrain_lookup[_terrain_lookup.keys()[0]] and is_instance_valid(_terrain_lookup[_terrain_lookup.keys()[0]])):
+	# check if terrain lookup is generated (dictionary and entries exists). If not -> generate
+	if not (_terrain_lookup and len(_terrain_lookup.keys()) > 0):
```

Comment:
I think the list has a count or size also. (Not sure which is better, len was from python)